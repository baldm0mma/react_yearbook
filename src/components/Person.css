.member_container {
  width: 22%;
  height: 300px;
  background-color: aliceblue;
  border-radius: 15px;
  margin: 20px;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.photo_container {
  width: 80%;
  overflow: hidden;
}

.photo {
  max-height: 200px;
  margin-top: 20px;
}

.name {
  color: purple;
  margin: 5px;
}

.quote {
  color: red;
  margin: 5px;
}

.superlative {
  color: green;
  margin-bottom: 15px;
}