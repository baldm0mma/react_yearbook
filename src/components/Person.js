import React from 'react';
import './Person.css';

const Person = props => {
  
  return (
    <div className='member_container'>
      <div className='photo_container'>
        <img className='photo' src={props.photo} alt='staff member' />
      </div>
      <div className='name'>
        {props.name}
      </div>
      <div className='quote'>
        "{props.quote}"
      </div>
      <div className='superlative'>
        {props.superlative}
      </div>
    </div>
  );
}

export default Person;
