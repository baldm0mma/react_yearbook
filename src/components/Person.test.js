import React from 'react';
import { shallow, mount } from 'enzyme';
import Person from './Person';

describe('Person', () => {
  it('should render the correct elements with the correct information', () => {
    const wrapper = shallow(<Person />);

    expect(wrapper).toMatchSnapshot();
  });
});