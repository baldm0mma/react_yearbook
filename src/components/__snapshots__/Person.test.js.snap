// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should render the correct elements with the correct information 1`] = `
<div
  className="member_container"
>
  <div
    className="photo_container"
  >
    <img
      alt="staff member"
      className="photo"
    />
  </div>
  <div
    className="name"
  />
  <div
    className="quote"
  >
    "
    "
  </div>
  <div
    className="superlative"
  />
</div>
`;
